src/iscsi.pyx

```diff
@@ -89,6 +94,9 @@ cdef class Task:
     def status(self):
         return scsi_task_get_status(self._task, NULL)
 
+    @property
+    def raw_sense(self):
+        return PyBytes_FromStringAndSize(&self._task.datain.data[2], self._task.datain.size-2)
```

Comment:
Small nit but python likes white space around left and right side of the "-" operand. Though....this is like hybrid C, anyways not required just mostly FYI.